airflow/api_fastapi/core_api/openapi/v1-generated.yaml

```diff
@@ -5056,7 +5056,7 @@ components:
       - git_version
       title: VersionInfo
       description: Version information serializer for responses.
-    airflow__api_fastapi__core_api__serializers__ui__dashboard__TaskInstanceState:
+    airflow__api_fastapi__core_api__datamodels__ui__dashboard__TaskInstanceState:
```

Comment:
Nit/future work: we should look at how we can change the name of this generated component

Reply:
Yup

Reply:
Solved in https://github.com/apache/airflow/pull/43824